package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Decomposition = Making smaller pieces of code - makes testing easier. Just as smaller sentences help us convey our intentions better.",
   "main": "transmitter.js",
   "scripts": {
-    "start": "echo \"Run the individual js files\" && exit 1"
+    "start": "echo \"Run the individual js files\" && exit 1",
+    "test": "mocha 'test/**/*.test.js'"
```

Comment:
Good addition of the command to test :)
Please change the github action workflow to run `npm test`, so it will run this instead of the original commands